### PR TITLE
Add component updater utility

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: Ashish Tiwari <ashishjaitiwari15112000@gmail.com>

**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: scripts/component_updation/main.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/layer5io/component_scraper/pkg"
+	"github.com/layer5io/meshkit/models/meshmodel/core/v1alpha1"
+)
+
+var (
+	ColumnNamesToExtract = []string{"Project Name", "Helm Chart", "Category", "Sub-Category", "Shape", "Primary Color", "Secondary Color", "Logo URL", "SVG_Color", "SVG_White"}
```
Comment (leecalcote) [inline scripts/component_updation/main.go:15]: @Revolyssup Please add a note in the spreadsheet that identifies the fact that columns are not to be renamed in the sheet without updating the code here.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Regarding the failing mesheryctl unit test - --- FAIL: TestProfileCmd/failing_add_authentication_test (0.00s)
[276](https://github.com/meshery/meshery/actions/runs/3565245982/jobs/5990159615#step:5:277) will you address, @Revolyssup?

**Comment by Revolyssup** [comment]
> Regarding the failing mesheryctl unit test - --- FAIL: TestProfileCmd/failing_add_authentication_test (0.00s) [276](https://github.com/meshery/meshery/actions/runs/3565245982/jobs/5990159615#step:5:277) will you address, @Revolyssup?

Yeah sure

**Comment by leecalcote** [comment]
@Revolyssup maybe I'm looking at stale test results. It seems that we're still failing the same test.

File: scripts/component_updation/main.go (hunk 1 above)
Comment (leecalcote) [reply scripts/component_updation/main.go:15]: We might want to incorporate use of an email on failure of this workflow.

**Pull request merged**